crates/bitwarden-wasm-internal/src/pure_crypto.rs

```diff
@@ -319,23 +317,59 @@ impl PureCrypto {
         Ok(result.to_encoded().to_vec())
     }
 
-    /// Given an encrypted private RSA key and the symmetric key it is wrapped with, this returns
+    /// Given a decrypted private RSA key PKCS8 DER and the symmetric key it is wrapped with, this returns
     /// the corresponding public RSA key in DER format.
     pub fn rsa_extract_public_key(
```

Comment:
note: breaknig change is fine here. This is not used yet. I will drop the clients PR replacing it with one using these functions.